src/sidebar/services/toast-messages.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * A service for managing toast messages.
+ */
+import { generateHexString } from '../util/random';
+
+const MESSAGE_DISPLAY_TIME = 5000;
+
+// @ngInject
+export default function toastMessagesService(store) {
+  const addMessage = (type, message) => {
```

Comment:
I think it may be useful to be able to override the MESSAGE_DISPLAY_TIME, is that easy to do?

I feel like the copy flash does not need to be as long as an error message.

Reply:
Perhaps display time could be variable, but I would want to calculate that within the service, and not make it a responsibility of the caller, ideally.

Reply:
Ya, that seems reasonable, and follows source of truth.